JiraPS/Private/Resolve-ErrorWebResponse.ps1

```diff
@@ -44,7 +44,29 @@ function Resolve-ErrorWebResponse {
             try {
                 $responseObject = ConvertFrom-Json -InputObject $responseBody -ErrorAction Stop
 
+                $errorArray = @()
                 foreach ($_error in ($responseObject.errorMessages + $responseObject.errors)) {
+
+                    foreach ($_err in $_error.PSObject.Properties.Value) {
+                        if ($_err -like "*You must specify a summary of the issue." -Or $_err -like "*is required.") {
```

Comment:
I don't trust parsing text from the answer. What happens when the system is installed in another language?
(Unfortunately I can't test that)

Is there any other way we can identify this?